vibi-dpu/src/core/review.rs

```diff
@@ -41,8 +42,9 @@ fn send_hunkmap(hunkmap_opt: &Option<HunkMap>, review: &Review) {
 	println!("HunkMap = {:?}", &hunkmap);
 	store_hunkmap_to_db(&hunkmap, review);
 	publish_hunkmap(&hunkmap);
-	// let hunkmap_async = hunkmap.clone();
-	// process_coverage(&hunkmap_async).await; TODO - include in future PR
+	let hunkmap_async = hunkmap.clone();
+	let review_async = review.clone();
+	process_coverage(&hunkmap_async, &review_async).await;
```

Comment:
We can add a status column to the hunks table in the database which we can update based on the success of this function for easier status report to the client.